### DISCUSSION TITLE
Automatically Split E2E Tasks (Atomizer) for component test

### DISCUSSION THREAD

**Discussion by YehudaGold** [nrwl/nx#27895] [Questions]
Hello,
I'm working on a large Nx project involving a React website and APIs. I have a significant number of Cypress component tests already written.

I've been exploring the new 'Automatically Split E2E Tasks (Atomizer)' feature and was eager to utilize it. However, after updating my Nx project, I've encountered an issue where the Atomizer doesn't seem to recognize my component test files.

I've reviewed the documentation and GitHub issues and understand that the Atomizer is currently designed to split Cypress e2e configerd tests, not component-test configerd tests.

I'm wondering if there's a potential workaround to use the Atomizer with Cypress component-tests.
Is there are plans to extend the Atomizer's functionality to include Cypress component-tests in the future? If so, are there any known challenges or limitations in implementing this feature?

Thanks.

**Comment by Tungsten78** [comment]
This was added in NX 21.6.1 - https://nx.dev/docs/technologies/test-tools/cypress/guides/cypress-component-testing#splitting-component-testing-tasks-by-file
